lib/centurion/service.rb

```diff
@@ -63,10 +63,11 @@ def image=(image)
       @image = image
     end
 
-    def build_config(server_hostname, &block)
+    def build_config(server_hostname, hostname = nil)
+
       container_config = {}.tap do |c|
         c['Image'] = image
-        c['Hostname'] = yield server_hostname if block_given?
+        c['Hostname'] = hostname if hostname
```

Comment:
That's correct. I tried to document this in the release notes: https://github.com/newrelic/centurion/releases/tag/v1.7.0